recipes/rotor/all/conanfile.py

```diff
@@ -86,11 +87,15 @@ def _configure_cmake(self):
         self._cmake = CMake(self)
         self._cmake.definitions["BUILD_BOOST_ASIO"] = self.options.enable_asio
         self._cmake.definitions["BUILD_THREAD"] = self.options.enable_thread
+        self._cmake.definitions["BUILD_THREAD_UNSAFE"] = not self.options.multithreading
         self._cmake.definitions["BUILD_TESTING"] = False
+        self._cmake.definitions["BUILD_SHARED_LIBS"] = self.options.shared
```

Comment:
```suggestion
```

The CMake helper does it automatically since the option `shared` exists. https://docs.conan.io/en/latest/reference/build_helpers/cmake.html#definitions